### PR TITLE
Add the `event` doc

### PR DESCRIPTION
**Pull request by AnnAngela**
### Description

> Add the `event` doc

---

### Checklist

- [ ] Does `npm test` pass with this change (including linting)?
- [ ] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

The PR only changes a doc.

Please help me to improve the writing, my English is so bad :(

### CODE REVIEW

**Comment by guyroyse** [comment]
Thanks for the contribution, @AnnAngela. Very helpful. I'd like to edit it a bit for grammar and spelling. Can you grant me access to do so? Thanks!

**Comment by AnnAngela** [comment]
Yes, I am very grateful for your willingness to help me!

File: README.md [hunk 1]
```diff
@@ -277,6 +277,20 @@ await Promise.all([
 
 Check out the [Clustering Guide](./docs/clustering.md) when using Node Redis to connect to a Redis Cluster.
 
+### Events
+
+Node Redis client class extends Node.js's `EventEmitter` class, and the client emits an event each time the network status changes:
```
Comment (guyroyse) [inline README.md:282]: The Node Redis client class is an `EventEmitter` and it emits an event each time the network status changes:

File: README.md [hunk 2]
```diff
@@ -277,6 +277,20 @@ await Promise.all([
...
+
+| Event name   | Scenes | Parameters |
```
Comment (guyroyse) [inline README.md:284]: Can we clean up the table format here to be a bit tidier?

Comment (guyroyse) [reply README.md:284]: I like to use this site: http://markdowntable.com/

File: README.md [hunk 3]
```diff
@@ -277,6 +277,20 @@ await Promise.all([
...
+| ------------ | ------ | ---------- |
+| connect      | The client is initiating a connection to the server. | _undefined_
+| ready        | The client successfully initiated the connection to the server. | _undefined_
+| end          | The client disconnected the connection to the server via `.quit()` or `.disconnect()`. | _undefined_
+| error        | When a network error has occurred, such as unable to connect to the server or the connection closed unexpectedly | The error object, such as `SocketClosedUnexpectedlyError: Socket closed unexpectedly` or `Error: connect ECONNREFUSED [IP]:[PORT]` |
+| reconnecting | The client is trying to reconnect to the server. | _undefined_
+
+Except for the network, the client will not emmit any events any more.
```
Comment (guyroyse) [inline README.md:292]: The client will not emit any other events beyond those listed abovee.

**Review by guyroyse — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (guyroyse) [reply README.md:292]: Doh! I spelled above wrong. ;)

**Comment by AnnAngela** [comment]
@guyroyse I'm sorry that I deleted the patch repository accidentally, so I have to open another pr #1753 to continue.

**Pull request closed without merging**